.github/workflows/dependabot-update-external-deps.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow is triggered when a pull request is opened by Dependabot
+# This workflow will use metadata from Dependabot update to update external_dependencies.txt and update the dependency across all the SDKs
+# This workflow will auto-approve and merge the pull request if the update is a patch version or minor version update with a compatibility score of 80 or higher.
+
+# NOTE: The auto-approved and merge step is currently disabled.
+name: Dependabot Update External Dependencies
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
+  repository-projects: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    steps:
+    - name: Fetch Dependabot metadata
+      id: dependabot-metadata
+      uses: dependabot/fetch-metadata@v2
+      with:
+        github-token: "${{ secrets.GITHUB_TOKEN }}"
+        alert-lookup: true
+        compat-lookup: true
+        skip-verification: true
+        skip-commit-verification: true
+
+    - name: Checkout repository
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        repository: ${{ github.event.pull_request.head.repo.full_name }}
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+
+    - name: Adjust External Dependencies
+      run: python eng/versioning/dependabot_update_external_dependencies.py --json '${{steps.dependabot-metadata.outputs.updated-dependencies-json}}'
+
+    - name: Commit changes
+      run: |
+        git config --global user.name 'github-actions[bot]'
```

Comment:
Correct me if I'm wrong but are you attempting to push new commits to the dependabot branch?